db/sqlinterfaces.c

```diff
@@ -2948,8 +2973,16 @@ static int skip_response_error(struct sqlclntstate *clnt)
 
 static int send_columns(struct sqlclntstate *clnt, struct sqlite3_stmt *stmt)
 {
-    if (clnt->osql.sent_column_data || skip_response(clnt))
+    if (clnt->osql.sent_column_data) {
+        logmsg(LOGMSG_DEBUG, "send_columns: Already sent column data\n");
         return 0;
+    }
+
+    int skip = skip_response(clnt);
+    if (skip) {
+        logmsg(LOGMSG_DEBUG, "send_columns: skip response = %d\n", skip);
```

Comment:
Don't need a 'skip' local variable. The function returns a boolean so you know here it can only be 0.